packaging/windows/packaging/create_install_wizard.iss

```diff
@@ -217,7 +228,9 @@ begin
     Exit;
   end;
 
-  PipArgs := Expandconstant('/c set "VIRTUAL_ENV={app}" && set "PATH={app}\Scripts;%PATH%" && pip install --pre --use-wheel --no-index --find-links . --force-reinstall --ignore-installed ' + mainPackageName);
+  PackagesToInstall := mainPackageName + ' cloudify-openstack-plugin cloudify-aws-plugin cloudify-softlayer-plugin cloudify-vsphere-plugin tosca-vcloud-plugin';
```

Comment:
i suggest you replace mainPackageName const with mainPackageNames and place all of the package names in it. will make this much clearer.